README.md

```diff
@@ -159,3 +159,33 @@ then 4 times in a row and issue an appropriate alert.
 [ES5]: https://docs.splunk.com/Documentation/SplunkCloud/9.0.2303/SearchReference/Transaction
 [ES6]: https://docs.splunk.com/Documentation/Splunk/9.0.4/SearchReference/Eval
 [ES7]: https://docs.splunk.com/Documentation/Splunk/9.0.4/SearchReference/JSONFunctions#json_object.28.26lt.3Bmembers.26gt.3B.29
+
+## Setting up a containerized Splunk instance
```

Comment:
To make things clearer, perhaps we should make a "running a test environment" section and then put this as a sub section under that.

Reply:
fixed